src/Microsoft.Framework.Runtime.Hosting/NamedResourceReader.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Framework.Runtime
+{
+    internal static class NamedResourceReader
```

Comment:
This is more like a _Helper_. The name _Reader_ make it sounds like an IO Stream Reader.